web-storage/README.md

```diff
@@ -0,0 +1,47 @@
+# Text
+
+Web Storage provides mechanisms to store key value pairs in a much more intuitive way than using cookies.
+
+There are two different APIs available:
+
+- sessionStorage
+
+  - The values are stored only for the duration of page session. The values get cleared when the tab is closed.
+
+  - Maximum of 5 MB can be stored.
+
+  #### Basic Usage
+
+  ```js
+  // Save data to sessionStorage
+  sessionStorage.setItem("name", "John Doe");
+
+  // Get saved data from sessionStorage
+  let name = sessionStorage.getItem("name");
+
+  // Remove saved data from sessionStorage
+  sessionStorage.removeItem("name");
+
+  // Remove all saved data from sessionStorage
+  sessionStorage.clear();
+  ```
+
+- localStorage
+
+  - localStorage is similar to sessionStorage, but it doesn't have an expiry.
```

Comment:
Also add few more points here, like:

- The keys and the values are always strings
- Data stored in localStorage, is never transferred to the server.
- Data is shared between all tabs and windows from the same origin.